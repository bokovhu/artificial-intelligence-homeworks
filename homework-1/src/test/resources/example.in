6 6
13
2 3
1 1
1 3
1 6
1 6
1 2
1 1
1 1
1 3
1 1
1 1
1 2
1 3