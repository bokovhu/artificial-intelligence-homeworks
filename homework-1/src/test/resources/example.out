1	1	1	1	4	4	6
1	1	1	1	4	4	6
2	2	3	3	4	4	7
2	2	5	5	4	4	7
2	2	5	5	4	4	7